libvdrtools/indy-wallet/src/wallet.rs

```diff
@@ -188,7 +189,9 @@ impl Wallet {
         );
 
         self.storage.add(&etype, &ename, &evalue, &etags).await?;
-        self.cache.add(type_, &etype, &ename, &evalue, &etags);
+        if cache_record {
+            self.cache.add(type_, &etype, &ename, &evalue, &etags);
+        }
```

Comment:
What is this change supposed to do? I see the flag is set to `true` in both instances where this is used.

Reply:
In migration, the flag is set to `false`.